src/tools/blast/components/results/BlastResultsButtons.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, lazy, useState, Suspense } from 'react';
+import { DownloadIcon, BasketIcon } from 'franklin-sites';
+import SidePanel from '../../../../shared/components/layouts/SidePanel';
+
+const BlastResultsButtons: FC<{
+  jobId: string;
+}> = ({ jobId }) => {
+  const BlastResultDownload = lazy(() =>
+    import(/* webpackChunkName: "blast-download" */ './BlastResultDownload')
+  );
+
+  const [displayDownloadPanel, setDisplayDownloadPanel] = useState(false);
+
+  return (
+    <>
+      {displayDownloadPanel && (
+        <Suspense fallback>
+          <SidePanel>
+            <BlastResultDownload
+              id={jobId}
+              onClose={() => setDisplayDownloadPanel(!displayDownloadPanel)}
```

Comment:
Why not simply set to `false`?

Reply:
In the first PR on side panel, @its-mahdi mentioned that the Download button can still be visible and we both thought you'd expect it to toggle the panel if you clicked on it again.

Reply:
Well, that's definitely OK for the other place where you call `setDisplayDownloadPanel` (the download button) but here the `onClose` is quite clear about the intent of what should be done within the function.

Reply:
Good point, renamed to `onToggleDisplay`